src/frontend/scss/generic/_background.scss

```diff
@@ -5,8 +5,16 @@ $gradient-colors: () !default;
 }
 
 // Build background colors
-@each $name, $value in $palette {
-  .bg-#{$name} {
-    background-color: $value !important;
+@each $name, $value in $colors {
+  @if type-of($value) == 'map' {
+    @each $sub-name, $sub-value in $value {
+      .bg-#{($name + '-' + $sub-name)} {
+        background-color: $sub-value !important;
+      }
+    }
+  } @else {
+    .bg-#{$name} {
```

Comment:
Colors utility classes are already generated by cunningham ( https://openfun.github.io/cunningham/?path=/story/getting-started-colors--page ) and can be imported via `@import "@openfun/cunningham-react/style";`